sdk-utils/src/main/java/com/redhat/parodos/sdkutils/WorkFlowServiceSdkUtils.java

```diff
@@ -36,9 +36,9 @@
  */
 
 @Slf4j
-public abstract class SdkUtils {
+public abstract class WorkFlowServiceSdkUtils {
```

Comment:
can we go with: `WorkFlowServiceUtils` ?
sdk is already implied by the package name.

Reply:
Updated. I also changed the NotificationServiceSdkUtils according to your suggestion.